wolfcrypt/src/wc_port.c

```diff
@@ -2178,6 +2189,221 @@ int wolfSSL_CryptHwMutexUnLock(void)
         return 0;
     }
 
+#elif defined(MAXQ10XX_MUTEX)
+    static pthread_mutex_t *wcCryptHwSharedMutexPtr;
```

Comment:
Section is not portable. Fine with me as long as this `MAXQ10XX_MUTEX` can be disabled. Also noticed not using our stdlib XMALLOC/XMEMSET, etc.

Reply:
`MAXQ10XX_MUTEX` is only enabled if configured with `--with-maxq10xx`